iib/workers/config.py

```diff
@@ -49,6 +49,7 @@ class DevelopmentConfig(Config):
     iib_log_level = 'DEBUG'
     iib_registry = 'registry:8443'
     iib_registry_credentials = 'iib:iibpassword'
+    iib_omps_url = 'https://omps-dev.cloud.paas.upshift.redhat.com/v2/{organization}/zipfile'
```

Comment:
If this value is not set and we are handling a legacy operator, then an exception should be raised before the build starts saying that OMPS is not configured and fail the request with that reason.